### PR TITLE
correct data in installation sql

### PR DESCRIPTION
**Pull request by brianteeman**
The sample_brochure.sql for sql azure had data from Joomla 2.5 - oops

### CODE REVIEW

**Comment by brianteeman** [comment]
@C-Lodder especially for you ;)
https://twitter.com/UKLodder/status/776709269366255616

File: installation/sql/sqlazure/sample_brochure.sql [hunk 1]
```diff
@@ -321,14 +321,13 @@ TRUNCATE TABLE [#__template_styles];
...
+SELECT 4, 'beez3', 0, 0, 'Beez3 - Default', '{"wrapperSmall":"53","wrapperLarge":"72","logo":"images\\/joomla_black.png","sitetitle":"Joomla!","sitedescription":"Open Source Content Management","navposition":"left","templatecolor":"personal","html5":"0"}'
 UNION ALL
-SELECT 3, 'atomic', 0, 0, 'Atomic - Default', '{}'
+SELECT 5, 'hathor', 1, 0, 'Hathor - Default', '{"showSiteName":"0","colourChoice":"","boldText":"0"}'
 UNION ALL
-SELECT 4, 'beez3', 0, 0, 'Beez3 - Default', '{"wrapperSmall":"53","wrapperLarge":"72","lo":"images/joomla_black.png","sitetitle":"Joomla!","sitedescription":"Open Source Content Management","navposition":"left","templatecolor":"personal","html5":"0"}'
+SELECT 7, 'protostar', 0, '1', 'My Default Style (Protostar)', '{"templateColor":"#696969","templateBackgroundColor":"#E3E3E3","logoFile":"","googleFont":"1","googleFontName":"Open+Sans","fluidContainer":"0"}'
 UNION ALL
-SELECT 5, 'hathor', 1, 0, 'Hathor - Default', '{"showSiteName":"0","colourChoice":"","boldText":"0"}'
+SELECT 8, 'isis', 1, '1', 'isis - Default', '{"templateColor":"#000000","logoFile":"","admin_menus":1,"displayHeader":1,"statusFixed":1,"stickyToolbar":1}'
```
Comment (brianteeman) [inline installation/sql/sqlazure/sample_brochure.sql]: thanks and done

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 159006809433ad47acd5af09120812706cf291dd<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12760">issues.joomla.org/tracker/joomla-cms/12760</a>.</sub>

**Comment by jeckodevelopment** [comment]
I have tested this item :white_check_mark: successfully on 159006809433ad47acd5af09120812706cf291dd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12760">issues.joomla.org/tracker/joomla-cms/12760</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12760">issues.joomla.org/tracker/joomla-cms/12760</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**